packages/x-date-pickers/src/locales/deDE.ts

```diff
@@ -80,13 +80,13 @@ const deDEPickers: Partial<PickersLocaleText<any>> = {
   dateTableLabel: 'Datum auswählen',
 
   // Field section placeholders
-  // fieldYearPlaceholder: params => 'Y'.repeat(params.digitAmount),
-  // fieldMonthPlaceholder: params => params.contentType === 'letter' ? 'MMMM' : 'MM',
-  // fieldDayPlaceholder: () => 'DD',
-  // fieldHoursPlaceholder: () => 'hh',
-  // fieldMinutesPlaceholder: () => 'mm',
-  // fieldSecondsPlaceholder: () => 'ss',
-  // fieldMeridiemPlaceholder: () => 'aa',
+  fieldYearPlaceholder: (params) => 'J'.repeat(params.digitAmount),
```

Comment:
@joserodolfofreitas I used the first letter of each word just like in english
If you can check that these placeholder make sense.

I'm using german in the doc because we use it in the example above.

Reply:
Honestly, I'm still not sure about this format localisation based on localised section name.
Did we have any benchmarks for introducing it?
I imagine some languages might have issues with it.
For example in Lithuanian:
- `Year` -> `Metai`
- `Month` -> `Mėnuo`
- `Day` -> `Diena`

I take it that based on other locales this would translate to `MMMM-MM-DD` (we're mostly using ISO format).
IMHO, I would find it utterly confusing. 🙈

Reply:
The common usage may depend on the language.
In French, it is very common to have the `AAAA` for the year.
Nothing forces the locale to use the language value, if in Lithuanian, people are used to the english values, nothing prevents from keeping them in the Lithuanian locale.

Reply:
Fair points. It will allow control over what a different locale users would expect.

But it still leaves the question of German locale and `ss:mm:ss` (hh:mm:ss)
I did not find a way to trigger our components into having the seconds in the field format, other than providing `format` (which overrides this behaviour).

That's also a question and maybe a feature we should support?
If `views` array contains `seconds`—maybe it should be included in the format? Especially given the fact, that time `views` currently have no effect on `Next` pickers 🤔

Reply:
> But it still leaves the question of German locale and ss:mm:ss (hh:mm:ss)

The best answer here would have to get a German native speaker (or at least better than me) :laughing: 

> I did not find a way to trigger our components into having the seconds in the field format, other than providing format (which overrides this behaviour).

There is no other way, we could indeed have a smart format based on the year on `TimeField` just like on `DateField`

Reply:
If you want, we can add a note to say that all locale decide to translate or not there placeholders.
If the doc I did felt confusing

Reply:
> If you want, we can add a note to say that all locale decide to translate or not there placeholders.
> If the doc I did felt confusing

Yup, I'd say that might end being useful. Basically mention that if field placeholder translations are not available for the selected locale—English ones will be used.

I'd also say that it might be useful to have a link to this section from `Field components` page.
Maybe add a section there, or just a `info` alert somewhere in the content?

Reply:
> I'd also say that it might be useful to have a link to this section from Field components page.

I am not sure how the `Field components` page should look like, its content should probably move in part to `Getting Started` and in part to a more advanced field page.
But until we have a clear vision, sure we can add a note there.

Reply:
@LukasTy Added